script/parse-examples

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+function checkout() {
+  repo=$1; url=$2; sha=$3
+
+  if [ ! -d "$repo" ]; then
+    git clone "https://github.com/$url" "$repo"
+  fi
+
+  pushd "$repo"
+  git fetch && git reset --hard "$sha"
+  popd
+}
+
+checkout examples/openscad openscad/openscad d7c5543dcadd7b60681305f9d1f8dc5de292f087
```

Comment:
I'd prefer to just have a git submodule as part of the repository.

Reply:
Many main tree-sitter org parsers use a script like this. Git submodules can be sucky, and add unnecessary burden on a user testing to add it.